### DISCUSSION TITLE
PHP version?

### DISCUSSION THREAD

**Discussion by jtibbles** [spatie/laravel-google-fonts#2] [Q&A] [answered]
This looks great - what PHP version does it need? I'm running 7.2 and couldn't install it.

**Comment by freekmurze** [comment, marked as answer]
PHP 8, if you need a lower version, feel free to fork the package.

**Reply by strarsis** [reply]
@freekmurze: Could I use it with PHP 7.x unmodified or would it surely not work out of the box?
